media/css/firefox/lockwise/lockwise.scss

```diff
@@ -6,52 +6,62 @@ $font-path: '/media/fonts';
 $image-path: '/media/protocol/img';
 
 @import '../../../protocol/css/includes/lib';
-@import '../../../protocol/css/components/hero';
 @import '../../../protocol/css/components/picto';
+@import '../../../protocol/css/components/split';
 @import '../../../protocol/css/templates/card-layout';
 @import '../../../protocol/css/templates/multi-column';
+@import '../../../protocol/css/components/logos/wordmark';
+@import '../../../protocol/css/components/logos/wordmark-product-lockwise';
 
+// split
 
-.add-on-download p {
-    margin: $spacing-sm;
+.mzp-c-split {
+    padding: $spacing-2xl;
 }
 
-.card-header {
-    text-align: center;
+.mzp-c-split-container {
+
+    p {
+        font-size: 18px;
+    }
+
+    @media #{$mq-lg} {
+        box-sizing: content-box;
```

Comment:
This is overriding the default box sizing values, but I notice the original also contains a bunch of vendor prefixed properties:

```css
-webkit-box-sizing: border-box;
-moz-box-sizing: border-box;
box-sizing: border-box;
```

Is it worth overriding the vendor prefixes here as well?

Reply:
This is a good point. I totally missed the prefixes when making my split updates, so leaving just the box-sizing property would be more consistent for the current status of bedrock. This should all be able to be cleaned out with the next protocol update in Bedrock where content-box sizing is the split default thanks to https://github.com/mozilla/protocol/pull/703/files

It's a well enough supported property that we'd only notice the missing prefixes on Safari (< 5.1), Chrome (< 10), and Firefox (< 29) (according to [CSS Tricks](https://css-tricks.com/box-sizing/#vendor-prefixes))

Reply:
Yep, this should only effect older browsers so only definitely a nice-to-have.